runner/engine.go

```diff
@@ -2,6 +2,7 @@ package runner
 
 import (
 	"fmt"
+	"github.com/pkg/browser"
```

Comment:
You are advised to change the dependency location according to the following rules:
1. Official dependence.
2. External dependencies
3. Internal dependencies